src/main/java/com/easyrest/components/admin/ActionOnClient.java

```diff
@@ -6,12 +6,12 @@
 import java.util.List;
 
 /**
- * This class is page component of Admin Panel that contains table with information about client and contains method
- * to make action on client - ban client or make him active.
+ * This class is component of Admin Panel that contains table with information about client and contains method
+ * to get first name from client list and make action on client - ban client or make him active.
  * Client - any type of user role (owner or administrator or admin or waiter or moderator or user).
  */
 
-public class ActionOnUser {
+public class ActionOnClient {
```

Comment:
Why change the name of this class?

Reply:
Because client - it is all users, there is class for all users